src/main/resources/swagger/api-docs.yaml

```diff
@@ -1475,6 +1500,27 @@ definitions:
       name:
         type: string
         description: Entity name
+  DeleteEntities:
+    properties:
+      recursive:
+        type: boolean
```

Comment:
until ui/orch/rawls supports recursion, I see you're setting this to false in the UI. Add "default: false" to this property definition to keep it aligned. False is probably the right default anyway.